prototype/link/scripts/MM_analyses/supervisor_Pilot.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+#
+# Automate the analysis of MM videos 
+# Assumes you are running from the subject base directory
+
+# Step 1: Z-score data while ignoring NaNs
+# Step 2: Align the participant data to adult standard 
+# Step 3: Extend the data if there are TRs missing at the end
+# Step 4: figure out which TRs they had their eyes closed for 
+# Transfers all of these files to the group folder
+#
+# TY 07112019 
+# Pilot updates TY 09132019
+# Reworked so everything can be run in participant folder TY 052021
+#
+#SBATCH --output=./logs/supervisor_MM_Pilot-%j.out
+#SBATCH -p day
+#SBATCH -t 1:00:00
+#SBATCH --mem 16000
+
+if [ $# -eq 0 ]
+then
+    analysis_type='default'
+else
+    analysis_type=$1
+fi
+
+preprocessing_type='linear_alignment' # could also be "nonlinear"
```

Comment:
Sometimes this is hard coded, sometimes this is an argument. COnsistency might be best. Also, should nonlinear be the default?